src/backend/assets/js/main.js

```diff
@@ -112,8 +112,49 @@ const loadVideoAndSlides = function(div, videoURL, slideURL) {
   }
 };
 
-window.handleClientLoad = handleClientLoad;
-window.initClient = initClient;
-window.handleAuthClick = handleAuthClick;
-window.listUpcomingEvents = listUpcomingEvents;
-window.loadVideoAndSlides = loadVideoAndSlides;
+$(document).ready(function () {
+  let filter;
+  //store room filters
+  let room ={
+    dateFilter:'.date-filter'
+  };
+  //store track filters
+  let track ={
+    dateFilter:'.date-filter'
+  };
+  //store schedule filters
+  let schedule={
+    dateFilter:'.day-filter'
+  };
+
+  const noResult=function(filterType,calendarMode) {
```

Comment:
Shouldn't there be a space after the `,` comma?

How about the policy of using a verb/action for function names? `showNoResults()`